lms/djangoapps/teams/static/teams/templates/instructor-tools.underscore

```diff
@@ -0,0 +1,13 @@
+<div class="wrapper-msg">
+    <span class="left-floater">
```

Comment:
I just noticed this in the email notification I got -- I think you can maybe XSS gmail by adding HTML tags in markdown code blocks?

`<script>alert("XSS")</script>`

<img width="888" alt="gmail-lol" src="https://cloud.githubusercontent.com/assets/1819831/9671132/ab4984b2-525e-11e5-847b-b9a779da6a1f.png">